subtests/docker_cli/attach/attach.py

```diff
@@ -95,7 +94,7 @@ def initialize(self):
         # Runs in background
         self.sub_stuff['cmdresult'] = dkrcmd.execute(run_in_pipe_r)
         self.sub_stuff['cmd_run'] = dkrcmd
-        self.wait_interactive_cmd()
+        dkrcmd.wait_for_ready(timeout=self.config['wait_interactive_cmd'])
```

Comment:
I'm okay if you want to just make ``wait_interactive_cmd`` a global default.  Seems it's used in more than one place.  I'm doubtful that being able to specify different timeout values for different tests is ever going to be useful :wink:

Reply:
I think this may conflict with #772. If 772 looks good, merge that, and I'll take a fresh look at this one afterward. Thanks.

Reply:
Yep, was reviewing PRs bottom-up so didn't know about that one yet :grin: .  Ya, I'd like to merge the other one since it updates all the tests, the name-difference isn't important to me.  But #772 needs a quick documentation/comment update.  After merging, fixing the conflict here should be really easy.